docker/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.6
```

Comment:
Any reason not to go to latest stable release? (3.9?)

Reply:
No, it is just my bias of being a bit conservetive with respect to new stuff 
Will update to 3.9